release-notes.md

```diff
@@ -58,6 +58,57 @@ it's available.
 In this section, we will cover historical information on
 past releases and how you can learn more.
 
+### 1.6.1 (2020-03-18)
+
+This maintenance release contains bugfixes since the 1.6.0 release. We deem it medium priority 
+for upgrading.
+
+In particular the fixes contained in this maintenance release address bugs in continuous aggregates, time_bucket_gapfill, 
+partial index handling and drop_chunks.
+
+For this release only, you will need to restart the database after upgrade before restoring a backup.
+
+**Minor Features**
+*  #1666 Support drop_chunks API for continuous aggregates
+*  #1711 Change log level for continuous aggregate materialization messages
+
+**Bugfixes**
+*  #1630 Print notice for COPY TO on hypertable
+*  #1648 Drop chunks from materialized hypertable
+*  #1668 Cannot add dimension if hypertable has empty chunks
+*  #1673 Fix crash when interrupting create_hypertable
+*  #1674 Fix time_bucket_gapfill's interaction with GROUP BY
+*  #1686 Fix order by queries on compressed hypertables that have char segment by column
+*  #1687 Fix issue with disabling compression when foreign keys are present
+*  #1688 Handle many BGW jobs better
+*  #1698 Add logic to ignore dropped chunks in hypertable_relation_size
+*  #1704 Fix bad plan for continuous aggregate materialization
+*  #1709 Prevent starting background workers with NOLOGIN
+*  #1713 Fix miscellaneous background worker issues
+*  #1715 Fix issue with overly aggressive chunk exclusion in outer joins
+*  #1719 Fix restoring/scheduler entrypoint to avoid BGW death
+*  #1720 Add scheduler cache invalidations
+*  #1727 Fix compressing INTERVAL columns
+*  #1728 Handle Sort nodes in ConstraintAwareAppend
+*  #1730 Fix partial index handling on hypertables
+*  #1739 Use release OpenSSL DLLs for debug builds on Windows
+*  #1740 Fix invalidation entries from multiple caggs on same hypertable
+*  #1743 Fix continuous aggregate materialization timezone handling
+*  #1748 Fix remove_drop_chunks_policy for continuous aggregates
+*  #1756 Fix handling of dropped chunks in compression background worker
+
+**Thanks**
```

Comment:
I don't know if you wanted this to be a list?  If so, you need to put an asterisk in front of each line.

Reply:
Done and pushed